tox.ini

```diff
@@ -69,7 +69,7 @@ max-complexity = 14
 commands =
     black --exclude "/(\.eggs|\.git|\.tox|\.venv|\.build|dist|charmhelpers|mod|files)/" .
 deps =
-    black
```

Comment:
I see this part is still pinned. Why is it necessary?

Reply:
Oh sorry I completely overlooked it. Thank you for noticing. It's fixed now.